### PR TITLE
  Interactions dialogue, all the controls for the interface, not still functioning. Waiting for implementation of factors.  

### PR DESCRIPTION
**Pull request by deaspo**

### CODE REVIEW

File: instat/dlgInteractions.vb [hunk 1]
```diff
@@ -1,3 +1,85 @@
...
+    ''' </summary>
+    ''' <param name="sender">The source of the event.</param>
+    ''' <param name="e">The <see cref="KeyEventArgs"/> instance containing the event data.</param>
+    ''' <code>
+    ''' autoTranslate(me) handles the multilingual capabilities for the dialogue.
+    ''' </code>
+    Private Sub dlgInteractions_KeyUp(sender As Object, e As KeyEventArgs) Handles Me.KeyUp
+        autoTranslate(Me)
+    End Sub
+
```
Comment (volloholic) [inline instat/dlgInteractions.vb]: Why are you doing this on Keyup?

**Comment by volloholic** [comment]
Hi Nice to see the attempt at inline documentation shall we discuss today.

File: instat/dlgInteractions.vb (hunk 1 above)
Comment (deaspo) [inline instat/dlgInteractions.vb]: My bad, changing it to load

**Comment by deaspo** [comment]
If possible, lets discuss at around 12:30pm

**Pull request closed without merging**